app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/owned/RequestedOwnedDetailActivity.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020. Book Friends Team
+ * All rights reserved.
+ * github URL: https://github.com/CMPUT301F20T21/Book_Friends
+ */
+
+package com.cmput301f20t21.bookfriends.ui.library.owned;
+
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.annotation.Nullable;
+
+import com.cmput301f20t21.bookfriends.ui.component.BaseDetailActivity;
+
+public class RequestedOwnedDetailActivity extends BaseDetailActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        button.setVisibility(View.GONE);
```

Comment:
Initially thought doing "View All Request" button, but there is one critical issue that "View All Request" activity might actually change the status of the book (to either Accepted or Available), then when returning from that activity this detail page is no longer in sync with the current book status. Can't think of easy solutions to solve this.. 

So the alternative solution is just to have no buttons there, "View All Request" can still be opened from the popup menu in the recyclerView

Reply:
Maybe on accept request success, we can finish  the current detail activity and start accepted detail activity?

Reply:
we also could try using a snapshot listener and re-render the button on return. that way we don't have to split the owned detail.

Reply:
A lot is different between the detail activity tho. Like available/requested detail doesn't have meeting location. Accepted has meeting location + waiting for borrower to scan state for the action button, and more.

Reply:
you are right. let's try the finishing activity way first if there's no other way. I didn't try it but wouldn't activity finish have a transition screen? like seeing the page popping off to the list and then pop back to the detail, just wondering. 

but imo we could still brute force and plug everything in. :) just have two larger rerender method and name the private func with more distinguishable names

Reply:
managed to achieve the expected behaviour with switching activities, and looks like `overridePendingTransition` is not needed